src/player_event_handler.rs

```diff
@@ -243,6 +243,10 @@ impl EventHandler {
                             );
                             env_vars.insert("FILTER", filter.to_string());
                         }
+                        // Ignore event irrelevant for standalone binary like PositionChanged
```

Comment:
So you comment here, that `PositionChanged` is ignored/irrelevant, but you changed `PositionCorrection` to `PositionChanged`? 

Could you take a look at the changes here again and check if that is as you wanted to commit it^^?

Reply:
Holy moly I am sorry. I removed the wrong block. Fixed it by removing the correct block and also removed the useless continue added by the autocomplete 🫣

Reply:
No worries and thanks for the quick fixup :)